crates/editor_ui/src/editor_tab.rs

```diff
@@ -14,15 +14,15 @@ pub trait EditorTab {
 #[derive(Clone, Hash, PartialEq, Eq, Debug, PartialOrd, Ord)]
 pub enum EditorTabName {
     CameraView,
+    EventDebugger,
     GameView,
     Hierarchy,
     Inspector,
+    Other(String),
```

Comment:
I'm really sorry, the `Other` was intentionally last because it is ordered after the other tabs, being user and community tabs.

Reply:
Fixed!